tests/test_helpers.py

```diff
@@ -85,15 +85,15 @@
      "afriend has been added/updated in contacts\n"
      ),
     (["helper", "u_contact", "Anthony", "--date", "2020-01-02"],
-     "Please choose from one of the following to update/add:\n"
+     "Please, rerun the helper with the same name as previously inputted,"
```

Comment:
How about "please rerun the helper specifying '-n list-index' to update item number 'list-index'"

Reply:
done!